drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c

```diff
@@ -1708,7 +1716,7 @@ static int bcm2835_isp_probe(struct platform_device *pdev)
 	if (ret) {
 		v4l2_err(&dev->v4l2_dev,
 			 "%s: failed to create ril.isp component\n", __func__);
-		goto error;
+		return ret;
```

Comment:
Unresolving as we will still leak the instance.

Either we do set ```*dev_int``` at the start so that ```bcm2835_isp_remove``` cleans up, or we need an error handler that calls ```bcm2835_isp_remove_instance```. The first is probably the easiest.

Reply:
Oops, I thought I did that!
Will set *dev_int on the top as it was in the first revision.